.gitmodules

```diff
@@ -18,3 +18,6 @@
 [submodule "scylla-tools"]
 	path = scylla-tools
 	url = ../scylla-tools-java
+[submodule "scylla-python3"]
+	path = scylla-python3
```

Comment:
Let's move it to tools/python3 instead (no need to repeat scylla). 

I also want to move scylla-jmx to tools/jmx and scylla-tools to tools/java.

Reply:
What about the build system changes?

```
$ git grep python3 configure.py
configure.py:#!/usr/bin/env python3
configure.py:arg_parser.add_argument('--python', action='store', dest='python', default='python3',
configure.py:          command = ./reloc/python3/build_reloc.sh
configure.py:          command = ./reloc/python3/build_rpm.sh
configure.py:          command = ./reloc/python3/build_deb.sh
configure.py:        build build/release/scylla-python3-package.tar.gz: build-python-reloc
configure.py:        build dist-python-rpm: build-python-rpm build/release/scylla-python3-package.tar.gz
configure.py:        build dist-python-deb: build-python-deb build/release/scylla-python3-package.tar.gz
```

Reply:
> Let's move it to tools/python3 instead (no need to repeat scylla).
> 
> I also want to move scylla-jmx to tools/jmx and scylla-tools to tools/java.
I think we can create another directory for these submodules, something like "packages".
ex) packages/jmx, packages/tools, packages/python3...
I suppose it's hard to understand "tools/java" is scylla-tools submodule , and also we use tools directory for other purpose, better to use other place.

Reply:
I think there won't be a problem. tools/ is a place for tools, like tools/toolchain, and we can add submodules and non-submodules there.